packages/postgres-database/src/simcore_postgres_database/models/workspaces.py

```diff
@@ -48,8 +54,11 @@
     ),
     column_created_datetime(timezone=True),
     column_modified_datetime(timezone=True),
+    column_trashed_datetime("workspace"),
```

Comment:
Q: should there not be a column tracking when it was trashed? A user or someone might be interested in this information.

Reply:
this column is exactly for that